### PR TITLE
38 cappi deviates from nominal altitude when multiple radars covers the same area

### PR DESCRIPTION
**Pull request by YngveE**
Code fixed so CAPPI composite are built correctly. The composites are used to create cartesian volumes in netcdf format. The volumes are used i Diana and as an input to osg 3D software to build a 3D volume from an image stack.

### CODE REVIEW

File: config/area_registry.xml
Comment (andershenja) [inline config/area_registry.xml:1]: area_registry.xml should not be updated with site-specific areas. Please revert these changes.

**Review by andershenja — CHANGES_REQUESTED** [review]

**Comment by IE-smhi** [comment]
I'll close this pull request and create a new branch where the reviews and changes are transparent and committed.

This is to make the history of changes easier to follow.
